### ISSUE TITLE
undefined: NewGrid

### ISSUE THREAD

**Issue by xeoncross** [andlabs/wakeup#8]
It seems that the grid struct is missing from the current andlabs/ui package.

**Comment by andlabs** [comment]
Yes; this program is still using the pre-libui version of andlabs/ui. Not sure if or when I'll update it.
